src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -27,7 +27,7 @@ const DragDropcontext: FC<dragDropProps> = ({
   refreshData,
 }) => {
   const [toogleSearch, setToogleSearch] = useState<boolean>(false);
-  const [isProcessing, setIsProcessing] = useState<boolean>(false);
```

Comment:
We can remove this entire line and occurrence of **setIsProcessing** since we will no longer need to use them